cfme/common/__init__.py

```diff
@@ -246,6 +248,9 @@ def __repr__(self):
     def load(self):
         self._keys = []
         key_values = []
+        if sel.is_displayed(self.MULTIKEY_LOC, root=self._entry):
+            # WE cannot process this kind of table yet.
+            return
```

Comment:
@psav 

No. It is a kind of table I haven't coded yet, so it just won't read it. This one will be a bit tricky to code up though.

FYI, the table has columns, first is name, second key, last value. So a sort of key/value database for that name. It is obvious that it does not fit the current scheme so a new class will have to be oded. But since it is not required yet ... :)